SCClassLibrary/Common/Control/Server.sc

```diff
@@ -38,8 +40,6 @@ ServerOptions {
 	var <>threads = nil; // for supernova
 	var <>useSystemClock = false;  // for supernova
 
-	var <numPrivateAudioBusChannels=112;
```

Comment:
Actually, I think I've found some hard linking that occurs here: 

https://github.com/supercollider/supercollider/blob/master/lang/LangPrimSource/OSCData.cpp#L983-L1072

But, everything past `inDevice`/`outDevice` should be fine.